benchmarks/GetParametersClassDefinitionWithMultipleConstructorParametersBench.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaminasBench\Di;
+
+use Laminas\Di\Definition\Reflection\ClassDefinition;
+use LaminasBench\Di\BenchAsset\ClassDefinitionWithMultipleConstructorParameters;
+
+/**
+ * @Revs(1000)
+ * @Iterations(20)
+ * @Warmup(2)
```

Comment:
We should probably declare these ourselves, rather than here - phpbench allows setting revs, warmup, iterations, etc in the CLI. 

Let's keep this minimal here 👍